meshtastic/__main__.py

```diff
@@ -638,6 +638,11 @@ def onConnected(interface):
 
         def setSimpleConfig(modem_preset):
             """Set one of the simple modem_config"""
+            channelIndex = our_globals.get_channel_index()
+            if channelIndex is not None and channelIndex > 0:
+                meshtastic.util.our_exit(
+                    "Warning: Cannot set modem preset for non-primary channel", 1
+                )
```

Comment:
We had a test expecting to enforce this, but weren't doing so. So if channel index is passed along with a modem preset, we expect it to be 0 now.